framework/core/hpp_instance.cpp

```diff
@@ -21,11 +21,23 @@
 #include <core/hpp_physical_device.h>
 #include <volk.h>
 
+#if defined(VKB_DEBUG) || defined(VKB_VALIDATION_LAYERS)
+#	define USE_VALIDATION_LAYERS 1
+#endif
+
+#if defined(USE_VALIDATION_LAYERS) && (defined(VKB_VALIDATION_LAYERS_GPU_ASSISTED) || defined(VKB_VALIDATION_LAYERS_BEST_PRACTICES) || defined(VKB_VALIDATION_LAYERS_SYNCHRONIZATION))
+#	define USE_VALIDATION_LAYER_FEATURES 1
+#endif
+
+#ifdef USE_VALIDATION_LAYERS
+#	define USE_VULKAN_LOGGER
```

Comment:
Is there a situation where you can enable `USE_VULKAN_LOGGER` without `USE_VALIDATION_LAYERS``? If not, do we need `USE_VULKAN_LOGGER` at all?

Reply:
See my above reply to @gpx1000 😄 

Although our use-case is tied to the validation layers, i didn't think it was a requirement that the loggers can only be used with the validation layers?

I was under the impression that any layer can hook into the logger and use it to communicate messages from a layer to an application? If that is the case, then there is value keeping this separate to demonstrate that the logger code is not exclusive to the layers

Reply:
Sorry, didn't saw that comment "thanks" to github's horrible UI.

In that case it makes sense, though I kinda fear that it may confuse people. A define that imply defines another define adds another layer, making code a bit less clear IMO. 

I'm fine having a dedicated logger define, but I think everything that's related to validation, which IMO also includes the debug callback, should be toggled by a define that tells people looking at the code that this is somehow related to validation.

Reply:
@SaschaWillems is there a specific change that you would like to see here? I can address this at the same time as @asuessenbach's latest comment

Reply:
Since there is no explicit use-case for `USE_VULKAN_LOGGER` we may as well remove that and keep it in mind for further discussions on that topic.

But tbh. the most important part is getting this merged with so many related issues coming up.